### PR TITLE
WHO 9 cards + 2 fixes

### LINKED ISSUE #3960: Kate Stewart incorrectly creates Human Soldiers instead of merely Soldiers
**Describe the bug**
Kate Stewart creates Human Soldier tokens instead of merely Soldiers.

**To Reproduce**
1. Play Kate Stewart.
2. Trigger her token-producing ability.
3. Observe.

**Expected behavior**
The tokens that Kate Stewart creates should create Soldiers, and nothing but Soldiers, just the same as Daddy Alistair (the Brigadier). Exactly why the tokens aren't _Human Soldiers_, like the ones from _Lord of the Rings_, I have no idea. Maybe some of those UNIT soldiers are secretly Zygons.

**Desktop (please complete the following information):**
 - OS: macOS Monterey Version 12.5.1
- Browser: Chrome
- Version: 1.6.59-SNAPSHOT-10.23

### PR DESCRIPTION
**Pull request by Simisays**
-closes #3960

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sally_sparrow.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Sally Sparrow
+ManaCost:2 W U
+Types:Legendary Creature Human Detective
+PT:2/3
+S:Mode$ CastWithFlash | ValidCard$ Creature | ValidSA$ Spell | Caster$ You | AffectedZone$ Exile,Graveyard,Hand,Library,Command | Description$ You may cast creature spells as though they had flash.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/sally_sparrow.txt:5]: AffectedZone not needed

File: forge-gui/res/cardsfolder/upcoming/the_cyber_controller.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Artifact Creature Cyberman
 PT:3/3
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill1 | TriggerDescription$ When CARDNAME enters the battlefield, each opponent mills X cards. Put all creature cards milled this way onto the battlefield face down under your control. They're 2/2 Cyberman artifact creatures.
 SVar:TrigMill1:DB$ Mill | Defined$ Opponent | NumCards$ X | RememberMilled$ True | SubAbility$ DBChangeZoneAll
-SVar:DBChangeZoneAll:DB$ ChangeZone | Origin$ Graveyard,Exile | GainControl$ True | Destination$ Battlefield | Defined$ Remembered | FaceDown$ True | FaceDownPower$ 2 | FaceDownToughness$ 2 | FaceDownSetType$ Artifact & Creature & Cyberman | SubAbility$ DBCleanup
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/the_cyber_controller.txt:7]: @tool4ever ChangeZone + `Defined$ Remembered.Creature` better ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/the_cyber_controller.txt:7]: Sure, I think this was already attempted here but lost the type in the refactor
@Simisays
you understand the difference?

File: forge-gui/res/cardsfolder/upcoming/sibyllin_soothsayer.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Sibylline Soothsayer
+ManaCost:2 R
+Types:Creature Human Warlock
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDigUntil | TriggerDescription$ Temporal Foresight — When CARDNAME enters the battlefield, reveal cards from the top of your library until you reveal a nonland card with mana value 3 or greater. Exile that card with three time counters on it. If it doesn’t have suspend, it gains suspend. Put the rest of the revealed cards on the bottom of your library in a random order.
+SVar:TrigDigUntil:DB$ DigUntil | Valid$ Card.nonLand+cmcGE3 | ValidDescription$ nonland permanent with mana value 3 or greater | FoundDestination$ Exile | RevealedDestination$ Exile | RememberFound$ True | SubAbility$ DBPutCounter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sibyllin_soothsayer.txt:6]: wrong RevealedDestination

File: forge-gui/res/cardsfolder/upcoming/sibyllin_soothsayer.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Sibylline Soothsayer
+ManaCost:2 R
+Types:Creature Human Warlock
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDigUntil | TriggerDescription$ Temporal Foresight — When CARDNAME enters the battlefield, reveal cards from the top of your library until you reveal a nonland card with mana value 3 or greater. Exile that card with three time counters on it. If it doesn’t have suspend, it gains suspend. Put the rest of the revealed cards on the bottom of your library in a random order.
+SVar:TrigDigUntil:DB$ DigUntil | Valid$ Card.nonLand+cmcGE3 | ValidDescription$ nonland permanent with mana value 3 or greater | FoundDestination$ Exile | RevealedDestination$ Library | RememberFound$ True | SubAbility$ DBPutCounter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sibyllin_soothsayer.txt:6]: random missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**